lib/ex_format.ex

```diff
@@ -720,16 +754,34 @@ defmodule ExFormat do
   end
 
   defp args_to_string(args, fun) do
+    args_to_string(args, fun, ", ")
+  end
+
+  defp args_to_string(args, fun, delimiter) do
     {list, last} = :elixir_utils.split_last(args)
     if last != [] and Inspect.List.keyword?(last) do
       prefix =
         case list do
           [] -> ""
-          _  -> Enum.map_join(list, ", ", &to_string(&1, fun)) <> ", "
+          _  -> Enum.map_join(list, delimiter, &to_string(&1, fun)) <> ", "
         end
-      prefix <> String.replace_suffix(kw_list_to_string(last, fun), ",\n", "")
+      kw_list_string =
+        # remove trailing comma and newline from multiline kw list args, if any
+        String.replace_suffix(kw_list_to_string(last, fun), ",\n", "")
+        |> handle_kw_list_delimiter(delimiter)
```

Comment:
I think [this rule](https://github.com/lexmag/elixir-style-guide#pipeline-operator) means this should be written line so:

```elixir
last
|> kw_list_to_string(fun)
|> String.replace_suffix(",\n", "")
|> handle_kw_list_delimiter(delimiter)
```